compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -0,0 +1,116 @@
+// Copyright 2020 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree-amd-aie/Transforms/KernelDispatch.h"
+
+#include "iree/compiler/Codegen/Utils/CPUUtils.h"
+#include "mlir/Dialect/MemRef/IR/MemRef.h"
+#include "mlir/Dialect/MemRef/Transforms/Transforms.h"
+#include "mlir/Transforms/GreedyPatternRewriteDriver.h"
+
+#define DEBUG_TYPE "kernel-dispatch"
+
+namespace mlir::iree_compiler::AMDAIE {
+
+/// Sets the lowering configuration for dispatch region with root op that
+/// implements the contraction operation interface.
+static LogicalResult setRootConfig(
+    func::FuncOp entryPointFn, linalg::ContractionOpInterface contractionOp) {
```

Comment:
Just make this `linalg::MatmulOp` for now. We can generalize it once we try to expand coverage.